rdmo/management/assets/js/components/elements/Views.js

```diff
@@ -37,11 +38,17 @@ const Views = ({ config, views, configActions, elementActions }) => {
                              options={getUriPrefixes(views)} />
           </div>
           {
-            config.settings.multisite && <div className="col-sm-4">
+            config.settings.multisite && <div className="col-sm-2">
               <FilterSite value={config.filter.views.sites} onChange={updateFilterSite}
                           options={config.sites} />
             </div>
           }
+          {
+            config.settings.multisite && <div className="col-sm-2">
+              <FilterEditor value={config.filter.views.editors} onChange={updateFilterEditor}
+                          options={config.sites} />
+            </div>
+          }
```

Comment:
This can be one block with a react fragment around it:

```
{
            config.settings.multisite && <>
               ...
            </>
}
```